src/Agent.Worker/CodeCoverage/CodeCoverageUtilities.cs

```diff
@@ -0,0 +1,137 @@
+﻿using Microsoft.VisualStudio.Services.Agent.Util;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.CodeCoverage
+{
+    public static class CodeCoverageUtilities
+    {
+        public const string RawFilesDirectory = "Code Coverage Files";
```

Comment:
I think that applies if we intend to have runtime constant instead of compile time. Here compile time constant fits much.